index.js

```diff
@@ -38,6 +42,9 @@ function start(func, options) {
   // Incoming requests to the readiness and liveness URLs
   server.register(healthCheck);
 
+  // Incoming requests to the metrics URL
+  server.register(metricsHandler(funcConfig.name, funcConfig.id, funcConfig.instance));
```

Comment:
Should probably just take the entire function config here instead of individual properties.